clippy_lints/src/try_err.rs

```diff
@@ -67,10 +68,15 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for TryErr {
 
             then {
                 let err_type = cx.tables.expr_ty(err_arg);
+                let span = if in_macro_or_desugar(err_arg.span) {
+                    span_to_outer_expn(err_arg.span)
```

Comment:
You could use `utils::snippet_with_macro_callsite` here (and `snippet` in the `else` block).

Reply:
Thanks. Done.